src/geckomat/utilities/saveEcModel.m

```diff
@@ -0,0 +1,69 @@
+function saveECmodel(model,modelAdapter,format,filename)
```

Comment:
Could you update tc0009 to test these three functions, i.e., load a conventional GEM, save it as ec, load it, and compare? Should be easy. Ideally the model should go through the processing chain so we test that metSmiles etc. are saved.